java/examples/pom.xml

```diff
@@ -36,6 +36,10 @@
       <groupId>org.apache.orc</groupId>
       <artifactId>orc-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.orc</groupId>
+      <artifactId>orc-shims</artifactId>
+    </dependency>
```

Comment:
CI seems to fail by this addition.

Reply:
I don't know why, but the current build prompt is the opposite of my environment, and I need some time to make sure to analyze it.